### ISSUE TITLE
Possible files for File Associations

### ISSUE THREAD

**Issue by esfox** [shalva97/kde-configuration-files#10]
https://github.com/shalva97/kde-configuration-files/blob/9200c0fc23247825f9536b89f8f83966aa6e4a1f/README.md?plain=1#L101

Hello there!

I was trying to actually look up which files might be the configuration for file associations and I found [this](https://unix.stackexchange.com/questions/164352/how-to-find-remove-file-associations-for-a-certain-program-in-kde).

I'll include here the important part of the link I provided as a quick reference.

> - $XDG_CONFIG_HOME/$desktop-mimeapps.list
>   - user overrides, desktop-specific (for advanced users)
> - $XDG_CONFIG_HOME/mimeapps.list
>   - user overrides (recommended location for user configuration GUIs)
> - $XDG_CONFIG_DIRS/$desktop-mimeapps.list
>   - sysadmin and ISV overrides, desktop-specific
> - $XDG_CONFIG_DIRS/mimeapps.list
>   - sysadmin and ISV overrides
> - $XDG_DATA_HOME/applications/$desktop-mimeapps.list
>   - for completeness, deprecated, desktop-specific
> - $XDG_DATA_HOME/applications/mimeapps.list
>   - for compatibility, deprecated
> - $XDG_DATA_DIRS/applications/$desktop-mimeapps.list
>   - distribution-provided defaults, desktop-specific
> - $XDG_DATA_DIRS/applications/mimeapps.list
>   - distribution-provided defaults

**Comment by chxei** [comment]
.config/mimeapps.list is possible the safest solution but although kde uses it, its not specifically related to kde. There is also lots of garbage in this file, for example my file contains:
`x-scheme-handler/jetbrains=jetbrains-toolbox.desktop
x-scheme-handler/msteams=teams.desktop
x-scheme-handler/postman=Postman.desktop`
these are obsolete for backing up because they are set automatically when you install application. 
But there are also useful lines, for example: `video/mp4=smplayer.desktop;`
So dunno if we should include these or not.

**Comment by chxei** [comment]
I was trying to set default browser and URL handler to portable ungoogled chromium. I have 
```
x-scheme-handler/http=/home/chxei/.local/share/applications/chrome.desktop;
x-scheme-handler/https=/home/chxei/.local/share/applications/chrome.desktop;
```
In my .config/mimeapps.list and some of the apps were respecting this settings(Konsole,discord...) but some of them don't(slack, steam). And apps that won't respect this config opened in Falkon(KDE's browser). So I was thinking wtf is going on?

So I remembered this issue and did some digging. First of all, on the cited link from @esfox, there is one line:
 "In this table, $desktop is one of the names of the current desktop, lowercase (for instance, kde, gnome, xfce, etc.)" 
So we have:
```
$XDG_CONFIG_HOME/kde-mimeapps.list
user overrides, desktop-specific (for advanced users)
$XDG_CONFIG_HOME/mimeapps.list
user overrides (recommended location for user configuration GUIs)
$XDG_CONFIG_DIRS/kde-mimeapps.list
sysadmin and ISV overrides, desktop-specific
$XDG_CONFIG_DIRS/mimeapps.list
sysadmin and ISV overrides
$XDG_DATA_HOME/applications/kde-mimeapps.list
for completeness, deprecated, desktop-specific
$XDG_DATA_HOME/applications/mimeapps.list
for compatibility, deprecated
$XDG_DATA_DIRS/applications/kde-mimeapps.list
distribution-provided defaults, desktop-specific
$XDG_DATA_DIRS/applications/mimeapps.list
distribution-provided defaults
```

I looked into them and many of them were absent. In files that weren't absent, there was no mention of Falkon. Then I found this on the arch wiki: "Applications describe what MIME types they can handle using desktop entries". I looked into org.kde.falkon.desktop and aha there was 
`MimeType=text/html;application/xhtml+xml;x-scheme-handler/http;x-scheme-handler/https;`
I added text/html and application/xhtml+xml entries in .config/mimeapps.list. nothing changed.
I added same MimeType attribute to chrome.desktop. nothing changed.
I removed falkon. nothing changed.
I copied chrome.desktop from .local/share/applications to /usr/share/applications. nothing changed.
I edited  /usr/share/applications/mimeapps.list and added chrome.desktop. nothing changed.
I edited /usr/share/applications/kde-mimeapps.list and added chrome.desktop. whuala now it is working. 

So conclusion: Its all messed up and configurations are all over the place. Additionally, Desktop entries themselves have an effect on it. So it is really hard to back up these files safely. @shalva97 adding these files to the configurations list is not acceptable from my point of view.
